voucher/rest.py

```diff
@@ -121,6 +162,43 @@ def use_vouchers(self, request, username=None):
         return Response([UseLogSerializer(p).data for p in pending_transactions], status=status.HTTP_201_CREATED)
 
 
+class RegisterLogViewSet(viewsets.ModelViewSet):
```

Comment:
Det er jo for å loggføre at man har solgt et kaffekort? Da holder det jo med CoffeeLog